_posts/2023-05-12-a-look-back-at-the-rebble-hackathon.md

```diff
@@ -0,0 +1,203 @@
+- [Alienware by Astosia](https://apps.rebble.io/en_US/application/637a6466c6c24a000a815e8b?section=watchface) 
+- [Time Twist Pop by Dook](https://apps.rebble.io/en_US/application/637a8d0cfdf3e30009f63964?section=watchfaces)
+- [Bloch Sphere by thyung](https://apps.rebble.io/en_US/application/6379e44cc6c24a000a815e83?section=watchfaces)
+- [Minimalin Reborn by vorsk](https://apps.rebble.io/en_US/application/63792649c6c24a000a815e70?section=watchfaces)
+- [TrekXmas by sethasaurus](https://apps.rebble.io/en_US/application/637921acfdf3e30009f63958?section=watchfaces)
+- [Alpine Wonderland by Kiwiscripter](https://apps.rebble.io/en_US/application/63780084c6c24a000a815e5a?section=watchfaces)
+- [Weezeface by John Spahr](https://apps.rebble.io/en_US/application/6377c82fc6c24a000a815e53?section=watchfaces)
+- [KemoPebble by CosmicTacoCat](https://apps.rebble.io/en_US/application/637aa7cbc6c24a000a815ea4?section=watchfaces)
+
+Each one of these watchfaces was made with love by an awesome member of our community, so thanks once more for helping keep our Pebbles alive!
+
+### Runner Up: Best Watchface
+
+This category was tricky, but after much deliberation the runner up for Best Watchface went to _Electric_ by Stefan Bauwens:
+
+![](/images/hackathon-look-back/ru_bwf.png)
+
+_Electric_ is a really neat watchface that displays the time and date on a nice graphic of a PCB. On the bottom left is what looks like an LED array. To make things even more impressive, you can pick what's displayed on this LED array and even automate it through tasker!
+It's a really nice touch to an already impressive looking watchface.
+
+Like all runners up & winners, Stefan received a fresh, boxed up Pebble watched signed by Eric Migicovsky! 
+
+![](/images/hackathon-look-back/stefan.png)
+
+### 🏆 Winner: Best Watchface
+
+The winner for best watchface was _Pixel_ by Nikki Murello! Pixel is an extensively customisable watchface that we felt captured people's love of the Pebble platform.
+
+![](/images/hackathon-look-back/w_bwf.png)
+
+Thanks to everyone who submitted a watchface, and please know that picking two winners was _so_ hard — all the submissions were fantastic!
+
+## Watch apps
+
+We decided to break the watch apps category in half for picking winners: Tools and Games. We received a wide number of tools, but before we get to the winners, let's look at all the watch apps across both categories that were submitted!
+
+![](/images/hackathon-look-back/apps.png)
+
+The quality of apps submitted during the hackathon blew me away. Watchapps can be tricky to make, and take a lot of time; it cannot be overstated just how impressive _all_ the submissions were. From top-left to bottom-right the submissions are:
+
+- [Services for Pebble by Blockarchitect - Manage your planning center services on your wrist](https://apps.rebble.io/en_US/application/6377f2c7fdf3e30009f63951)
```

Comment:
My name is slightly wrong here 😅

It's supposed to be a play on words thing but turns out it just makes it impossible for other people to type ha

Reply:
I so nearly got through it all without misspelling anyone's name 😁